webapi/src/main/scala/org/knora/webapi/OntologyConstants.scala

```diff
@@ -816,7 +816,6 @@ object OntologyConstants {
         val LinkValueHasSourceIri: IRI = KnoraApiV2PrefixExpansion + "linkValueHasSourceIri"
 
         val FileValueAsUrl: IRI = KnoraApiV2PrefixExpansion + "fileValueAsUrl"
-        val FileValueIsPreview: IRI = KnoraApiV2PrefixExpansion + "fileValueIsPreview"
```

Comment:
With IIIF, it doesn't make sense to have a separate preview image. If you want a preview, you just request the same image in a smaller size. The separate preview image in the old SALSAH was needed only because we didn't have IIIF yet.